Source/HTTP/WinHttp/winhttp_http_task.cpp

```diff
@@ -362,6 +505,30 @@ void winhttp_http_task::callback_status_request_error(
     }
 }
 
+
+#if HC_PLATFORM == HC_PLATFORM_GDK
+void winhttp_http_task::callback_status_sending_request(
+    _In_ HINTERNET hRequestHandle,
+    _In_ winhttp_http_task* pRequestContext,
+    _In_ void* /*statusInfo*/)
+{
+    if (hRequestHandle != nullptr)
+    {
+        HRESULT hr = XNetworkingVerifyServerCertificate(hRequestHandle, pRequestContext->m_securityInformation);
+        if (FAILED(hr))
+        {
+            pRequestContext->complete_task(hr, hr);
```

Comment:
>hr, hr [](start = 43, length = 6)

intentionally "hr" for both error values?

Reply:
Yeah, I wanted this hr to flow up so its obvious to the caller the cause.  Probably won't happen too often